src/test/java/teammates/test/cases/action/AdminEmailComposeSaveActionTest.java

```diff
@@ -77,7 +79,8 @@ public void testExecuteAndPostProcess() throws Exception {
                 Const.ParamsNames.ADMIN_EMAIL_ADDRESS_RECEIVERS, receiver);
         pageResult = getShowPageResult(action);
         assertEquals(
-                Const.ViewURIs.ADMIN_EMAIL + "?error=true&user=admin.user", pageResult.getDestinationWithParams());
+                getPageResultDestination(Const.ViewURIs.ADMIN_EMAIL, true, adminUserId),
```

Comment:
This should be like this ?
 
     assertEquals(
              getPageResultDestination(
                     Const.ViewURIs.ADMIN_EMAIL,
                     true,
                     adminUserId),